account_global_discount/views/account_invoice_views.xml

```diff
@@ -65,7 +66,7 @@
                     name="invoice_global_discount_ids"
                     nolabel="1"
                     readonly="1"
-                    attrs="{'invisible': [('global_discount_ids', '=', [])]}"
+                    invisible="global_discount_ids"
```

Comment:
Is this right? The equivalent expression should be the opposite one, if I'm not wrong

Reply:
You’re absolutely right, thank you for pointing this out. I’ve fixed it now